nav2_recoveries/plugins/back_up.cpp

```diff
@@ -16,25 +16,29 @@
 #include <ctime>
 #include <memory>
 
-#include "nav2_recoveries/back_up.hpp"
+#include "back_up.hpp"
 
 using namespace std::chrono_literals;
 
 namespace nav2_recoveries
 {
 
-BackUp::BackUp(rclcpp::Node::SharedPtr & node, std::shared_ptr<tf2_ros::Buffer> tf)
-: Recovery<BackUpAction>(node, "BackUp", tf)
+BackUp::BackUp()
+: Recovery<BackUpAction>()
```

Comment:
simulate_ahead_time{2.0}

Reply:
Never mind, I saw spin has a bunch of variables as well, so just declare on the body for consistency.

Reply:
its set in the constructor https://github.com/ros-planning/navigation2/blob/8b8985ec6165606f75b5edb496db95ad67c0527a/nav2_recoveries/plugins/back_up.cpp#L29